src/components/feedback/feedback.js

```diff
@@ -308,16 +310,25 @@ FeedbackOverlimitWarning.propTypes = {
 };
 
 Feedback.propTypes = {
-  type: PropTypes.string, // type of content the user is a evaluating
-  site: PropTypes.string.isRequired, // the site name, same value as the `site` value passed to ReactPageShell
-  section: PropTypes.string, // name of section the feedack component falls under
-  location: PropTypes.object.isRequired, // this prop is generated by batfish and provides context for the current page
+  /** The type of content the user is a evaluating */
```

Comment:
While I'm here: fixing the formatting of these props so they appear on the catalog page.